README.md

```diff
@@ -293,4 +293,31 @@ nix build github:DeterminateSystems/nix-installer#nix-installer.doc
 firefox result-doc/nix-installer/index.html
 ```
 
-[systemd]: https://systemd.io
\ No newline at end of file
+## Diagnostics
+
+The goal of the Determinate Nix Installer is to successfully and correctly install Nix.
+The `curl | sh` pipeline and the installer collects a little bit of diagnostic information to help us make that true.
```

Comment:
This sentence is a bit awkward. I'd suggest:

When you use the `curl | sh` method for running the installer, the server at `install.determinate.systems` collects some high-level diagnostic information that helps us improve the installer.